js_tests/admin/DateTimeShortcuts.test.js

```diff
@@ -40,3 +40,9 @@ QUnit.test('time zone offset warning', function(assert) {
     $('body').attr('data-admin-utc-offset', savedOffset);
     assert.equal($('.timezonewarning').text(), 'Note: You are 1 hour behind server time.');
 });
+
+QUnit.test('Calendar regular expression callback', function(assert) {
+    let format = "\n February \n 12 \n 1998 \r \r \r \t \t \t \\ \\ ' '";
```

Comment:
This is not really a format. I'd use sth more realistic:
```suggestion
    let date_input_format = "'%d \r\t\\\n %B \n\\\r\t %Y'";
```